src/index.js

```diff
@@ -1,3 +1,5 @@
 export { default } from './request';
 export { default as HttpError } from './HttpError';
 export { default as NetworkError } from './NetworkError';
+export { default as method } from './httpMethods';
+export { default as status } from './httpStatuses';
```

Comment:
Not 100% sure these names are quite right. Maybe:
`httpMethod` and `httpStatus`?

Reply:
I think it's fine as is, tbh. HTTP is implied with this library.

Reply:
I'm having a debate with myself about whether we should pluralise or not, though. I could go either way, and it's not plural as is, so might just be fine. But felt worth mentioning.

Reply:
I had the same debate! I went this way on the basis that we use `status` in Django so we get some consistency.

Reply:
Also seems more human readable to say `status.HTTP_200_OK` compared to `statuses.HTTP_200_OK`

Reply:
Yeah, agreed, those are all good reasons.